src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -641,8 +641,8 @@ private Path discoverSdkPath() {
               + ": exception thrown when searching for Google Cloud SDK", ex);
         }
       }
-      throw new AppEngineException("Google Cloud SDK path was not provided and could not be"
-          + " found in any known install locations.");
+      throw new CloudSdkNotFoundException("The Google Cloud SDK could not be found in the default"
+          + " locations and no path was provided.");
```

Comment:
Not sure if _default_ is correct here as the different resolvers  may have searched in more than the default locations.

_The Google Cloud SDK could not be located_?